projects/plugins/boost/app/assets/src/js/utils/generate-critical-css.ts

```diff
@@ -205,11 +228,31 @@ async function generateForKeys(
 					passthrough,
 				} );
 
+				// Track showstopper Critical CSS generation error.
+				const eventProps = {
+					time: Date.now() - startTime,
+					provider_key: providerKey,
+					error_message: err.message,
+					error_type: err.type,
```

Comment:
Show-stopper errors won't be a `SuccessTargetError`, and may or may not be a `UrlError` subclass, so `type` may not be populated.

In this context, can I suggest something like:
```suggestion
					error_type: err.type || err.constructor && err.constructor.name || 'unknown'
```

(Only applies to show-stoppers; URL errors are guaranteed to have a `type` to use)